loops-arrays-objects.js

```diff
@@ -9,11 +9,30 @@ const scores = [
     {name: 'Rianne', score: 66, grade: null}
 ];
 
+for (let i = 0; i < scores.length; i++){
+    if (scores[i].score < 60){
+        scores.grade = "E"
+    }
+    else if (scores[i].score < 70){
+        (scores[i].grade = "D")
+    }
+    else if (scores[i].score < 80){
+        scores[i].grade = "C"
+    }
+    else if (scores[i].score < 90){
+        scores[i].grade = "B"
+    }
+    else if (scores[i].score < 100){
+        scores[i].grade = "A"
+    }
+    console.log("name: "+scores[i].name+" score: "+scores[i].score+" grade: "+scores[i].grade);
+}
```

Comment:
dit was voldoende geweest: 
for (let i = 0; i < scores.length; i++) {
    console.log(scores[i].score);

Reply:
ik zie dat je alle opdrachten in opgave 1 hebt gebundeld in opgave 1a.